source/unified-test-format/unified-test-format.rst

```diff
@@ -1044,6 +1044,12 @@ non-negative integer:
 - ``<major>.<minor>`` (``<patch>`` is assumed to be zero)
 - ``<major>`` (``<minor>`` and ``<patch>`` are assumed to be zero)
 
+Any component other than ``major``, ``minor``, and ``patch`` MUST be discarded
```

Comment:
Consider whether to be more proscriptive of how the actual server version is obtained.  If the spec says explicitly that it's obtained by taking the first three elements of the `versionArray` field in the `buildInfo` reply, we can avoid any mention of discarding other components.

Reply:
I've expanded the wording to explain how to obtain this information from the `versionArray` field.